homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -906,6 +906,8 @@ async def speech_to_text(
                 metadata,
                 self._speech_to_text_stream(audio_stream=stream, stt_vad=stt_vad),
             )
+        except (asyncio.CancelledError, TimeoutError):
+            raise  # expected
```

Comment:
Since when do we expect this?

Reply:
Audio streams can time out, like when the user hangs up the phone in VoIP. , The pipeline task itself can also be cancelled if the integration is unloaded in the middle of a pipeline run.
In both cases, these don't need to be logged as errors.

Reply:
But why do we expect this after this PR?

Reply:
Oh, because VoIP is using the new utility function `queue_to_iterable` instead of manually wrapping the audio stream. So the exception will get bubbled up through `assist_pipeline` now.